### PR TITLE
screenreader now capable of reading checkbox group

### PR DESCRIPTION
**Pull request by ajspotts**
Resolves #{12123}

### Description
Enables users who utilize assistive technology to identify the name of the group ("Special Issues") that the checkboxes are located in by adding elements `<fieldset>` & `<legend>` identified by W3C standards as readable by a screenreader

### CODE REVIEW

File: client/app/containers/EstablishClaimPage/EstablishClaimDecision.jsx [hunk 1]
```diff
@@ -187,22 +198,27 @@ export class EstablishClaimDecision extends React.Component {
...
-                  label={issue.node || issue.display}
-                  name={issue.specialIssue}
-                  onChange={handleSpecialIssueFieldChange(issue.specialIssue)}
-                  key={index}
-                  value={specialIssues[issue.specialIssue]}
-                />;
-              })
-            }
-          </div>
+          <fieldset style={{ width: '100%' }}>
```
Comment (kevmo) [inline client/app/containers/EstablishClaimPage/EstablishClaimDecision.jsx]: Styling should probably be put into a variable elsewhere, then injected via that variable.

File: client/app/containers/EstablishClaimPage/EstablishClaimDecision.jsx [hunk 2]
```diff
@@ -99,83 +105,88 @@ export class EstablishClaimDecision extends React.Component {
...
+          {this.hasMultipleDecisions() && (
+            <div>
+              <h2>Select a Decision Document</h2>
+              <p>
+                Use the tabs to review the decision documents below and select
+                the decision that best fits the VACOLS Decision Criteria.
+              </p>
+              <TabWindow tabs={tabs} onChange={this.onTabSelected} />
+            </div>
+          )}
```
Comment (kevmo) [inline client/app/containers/EstablishClaimPage/EstablishClaimDecision.jsx]: Is everything above this line just formatting? Was having a hard time seeing any other differences.

Comment (ajspotts) [reply client/app/containers/EstablishClaimPage/EstablishClaimDecision.jsx]: Yes, had to re-formate after initial commit to undo changes from Prettier extension hence all the changes.  Sorry for the confusion!

**Comment by alisan16** [comment]
Looks awesome!

**Pull request merged**